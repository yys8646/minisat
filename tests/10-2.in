10
.0.0......
....1...1.
0.1.....10
.1...1....
...0.11...
...0......
....1...1.
0......11.
...1..0...
1....0....
