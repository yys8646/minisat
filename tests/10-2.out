10
1010011001
1001100110
0110100110
0101011001
1010011010
0110100101
1001101010
0110010110
0101010101
1001101001
